app/views/users/show.html.erb

```diff
@@ -248,6 +248,11 @@
 <div class="richtext text-break"><%= @user.description.to_html %></div>
 
 <% if @heatmap_data.present? %>
+  <div class="row">
+    <div class="col text-body-secondary fs-5">
+      <%= t(".contributions", :count => @changes_count) %>
+    </div>
+  </div>
```

Comment:
You don't need `row`/`col` here, you need `<h2>`.

(heatmap probably also doesn't need `row`/`col`)